pom.xml

```diff
@@ -90,7 +90,13 @@
   </distributionManagement>
 
   <dependencies>
-    <dependency>
+		<dependency>
```

Comment:
Looks like there's a mix of space and tab indent in this file.

Reply:
yes, do we want to fix that? (formatting that passes CI can be considered acceptable or else strengthen CI)